dspace-api/src/main/java/org/dspace/content/virtual/Collected.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content.virtual;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class Collected implements VirtualBean {
+
+    @Autowired
+    private ItemService itemService;
+
+    /**
+     * The fields for which the metadata will be retrieved
+     */
+    private List<String> fields;
+
+    /**
+     * Generic getter for the fields property
+     * @return The list of fields to be used in this bean
+     */
+    public List<String> getFields() {
+        return fields;
+    }
+
+    /**
+     * Generic setter for the fields property
+     * @param fields    the list of fields to which the fields property will be set to
+     */
+    public void setFields(List<String> fields) {
+        this.fields = fields;
+    }
+
+
+    public List<String> getValues(Context context, Item item) {
+        List<String> resultValues = new LinkedList<>();
+        List<String> value = this.getFields();
+        for (String s : value) {
+            String[] splittedString = s.split("\\.");
```

Comment:
This `\\.` split string looks to be hardcoded into at least two different places (see also `ItemServiceImpl` changes).  So, it should be made into a constant somewhere where it can be reused.

Reply:
This is the functionality to split a single-string metadata field to schema, element, qualifier.
I believe this happens in various locations in DSpace already, but I'm not sure why it should become a constant
The `MetadataField` class could be extended to support easily switching from e.g. dc.description.abstract to the separate schema, element, qualifier. But that sounds like something not specific to entities

Reply:
@benbosman and @AlexanderS : thanks for the additional context.  I agree we should just leave this as-is and resolve it once we find a solution for all of DSpace's API.  I've resolved this conversation.